opta/commands/apply.py

```diff
@@ -358,8 +361,26 @@ def _verify_parent_layer(layer: Layer) -> None:
                 "Please fix these issues and try again!"
             )
     except MissingState as e:
-        raise MissingState(
-            f"Failed to get the Environment state {e.args[0]}"
-            "Usually, this means that the Environment mentioned in configuration file does not exist."
-            "You can read more about this in our getting started guide: https://docs.opta.dev/getting-started/"
+        click.confirm(
+            f"Failed to get the Environment state {e.args[0]} "
+            "Usually, this means that the Environment mentioned in configuration file does not exist. \n"
+            "Would you like to create your environment?",
+            abort=True,
         )
+        t = tempfile.mkdtemp()
+        try:
+            config_path = os.path.join(t, "opta.yml")
+            with open(config_path, "w") as f:
+                f.write(layer.parent.original_spec)
```

Comment:
Why can't we just provide the path to the env file to `_apply`? Instead of creating this temp copy.